package.json

```diff
@@ -34,6 +34,8 @@
     "integration-sentinel-local": "export VERSION=$(node ./scripts/build/get-version.js) && ./scripts/build/build-service-images.sh && npm run ci-integration-sentinel",
     "integration-api": "cd api && npm ci && cd .. && mocha 'api/tests/integration/**/*.js' -t 10000",
     "lint": "eslint --color --cache . && ./scripts/build/blank-link-check.sh",
+    "lint-fix": "eslint --color --cache . --fix",
+    "standard-wdio-local": "export VERSION=$(node ./scripts/build/get-version.js) && ./scripts/build/build-service-images.sh && wdio run ./tests/e2e/standard/wdio.conf.js --suite=all",
```

Comment:
Can we make this change in a separate PR?

Reply:
Sure